build.gradle.kts

```diff
@@ -24,7 +24,7 @@ dependencies {
 
   implementation("uk.gov.justice.service.hmpps:hmpps-sqs-spring-boot-starter:5.4.4")
   implementation("io.opentelemetry:opentelemetry-api:1.50.0")
-  implementation("io.opentelemetry.instrumentation:opentelemetry-instrumentation-annotations:2.15.0")
+  implementation("io.opentelemetry.instrumentation:opentelemetry-instrumentation-annotations:2.16.0")
```

Comment:
in dependabot's #331 pr there is also

`testImplementation("io.swagger.parser.v3:swagger-parser:2.1.28")` 
do we need it?

and `kotlin("jvm") version "2.1.21"` in the other build.gradle.kts file

Reply:
the only problem **that i know of** with renovate's automatic prs is that 
`testRuntimeOnly("org.junit.platform:junit-platform-launcher:1.11.4")`
must be kept at that version